rest-api/model/participant_summary.py

```diff
@@ -47,6 +47,7 @@ class ParticipantSummary(Base):
   phoneNumber = Column('phone_number', String(80))
   loginPhoneNumber = Column('login_phone_number', String(80))
   email = Column('email', String(255))
+  primaryLanguage = Column('primary_language', String(80))
   recontactMethodId = Column('recontact_method_id', Integer, ForeignKey('code.code_id'))
   languageId = Column('language_id', Integer, ForeignKey('code.code_id'))
```

Comment:
We at a minimum should deprecate this field; we may be able to drop the column from the database and API entirely. We should ask HPRO folks about it.

Reply:
Let's add a comment saying this is deprecated?

Reply:
done